client/api/user.js

```diff
@@ -30,4 +32,29 @@ router.put("/:id", authMiddleware, async (req, res) => {
   return res.json({ user: newUser });
 });
 
+router.post("/:resume", authMiddleware, async (req, res) => {
```

Comment:
what's the `:` for before the resume? since it's a param, isn't this the behavior for any post route now? (unless i'm reading this wrong)

Reply:
I'm dumb, yeah you're right lol